container/Dockerfile

```diff
@@ -37,19 +40,11 @@ RUN echo -e 'startup_message off \nhardstatus on \nhardstatus alwayslastline \nv
 # Setup vim 
 RUN echo -e 'syntax enable \nset tabstop=4 \nset number \nset hlsearch \nset incsearch' > ${HOME}/.vimrc
 
-# Setup spark/scala/python/rest
-ENV SPARK_HOME="/spark_home"
-ENV PATH=$PATH:$SPARK_HOME/bin:$SPARK_HOME/sbin
-RUN ${SETUP_AREA}/setup-java.sh
-RUN ${SETUP_AREA}/setup-spark.sh
-RUN ${SETUP_AREA}/setup-scala.sh
-RUN ${SETUP_AREA}/setup-python.sh
-RUN ${SETUP_AREA}/setup-kafka.sh
-RUN ${IOT_REPO_PATH}/rest/install.sh
-
-# Copy ES 2.0.1.0 client libraries
-RUN echo "Downloading Event Store 2.0.1.0 client libraries, it will take some time..."
-RUN wget -P ${SPARK_HOME}/jars/ https://repo1.maven.org/maven2/com/ibm/event/ibm-db2-eventstore-client-spark-2.4.6/2.0.1.0/ibm-db2-eventstore-client-spark-2.4.6-2.0.1.0.jar
+# copy json file to user_volume
```

Comment:
Can you amend this comment as this is an important line:

```
# Copy the json/version file to the user_volume directory so it can be read by setup-container.sh.
# It is important to note that the location of this file (i.e. es-release.json) is expected to be in the same
# directory as this script (i.e. build.sh) if that ever changes this will need to be updated.